leehyowonzero/4week/43163.py

```diff
@@ -0,0 +1,33 @@
+from collections import deque
+
+def transpossible(a, b):
+    cnt = 0 
+    if(len(a) != len(b)):
+        return False
+    for i in range(len(a)):
+        if(a[i] != b[i]):
+            cnt += 1
```

Comment:
```suggestion
    for achar, bchar in zip(a, b):
        if achar != bchar:
            cnt += 1
```

`zip`을 이용해 표현할 수도 있습니다.